controllers/mapController.js

```diff
@@ -110,7 +115,87 @@ const addInvitedUser = async (req, res, next) => {
   }
 };
 
+const inviteNewMember = async (req, res, next) => {
+  const { id } = req.params;
+  const { email } = req.body;
+
+  try {
+    const currentMap = await Map.findById(id).exec();
+    const newUser = await User.findOne({ email }).exec();
+
+    const userId = newUser._id;
+    const verifyInvitedEmail = currentMap.invitationList
+      .map(user => user.email)
+      .some(invitedEmail => invitedEmail === email);
```

Comment:
추가적으로 이 부분을 아래처럼 작성하면 조금 더 직관적일 것 같다는 생각도 드는데
서영님, 성훈님 의견은 어떠신가요-?

```suggestion
    const verifyInvitedEmail = currentMap.invitationList
      .map(user => user.email)
      .includes(email);
```

Reply:
좋은 것 같아요 💡 감사합니당!

Reply:
더 직관적인 것 같아서 좋은 것 같아요~!